openlibrary/macros/Pager.html

```diff
@@ -19,17 +19,17 @@
     <div class="clearfix"></div>
     <div class="pagination">
     $if page != 1:
-        <a href="$changequery(page=None)" class="ChoosePage">&laquo;&nbsp;$_('First')</a>
-        <a href="$changequery(page=page-1)" class="ChoosePage">&lt;&nbsp;$_('Previous')</a>
+        <a href="$changequery(page=None)" class="ChoosePage" data-ol-link-track="Pager|First">&laquo;&nbsp;$_('First')</a>
+        <a href="$changequery(page=page-1)" class="ChoosePage" data-ol-link-track="Pager|Previous">&lt;&nbsp;$_('Previous')</a>
     $for p in range(first_page_in_set, last_page_in_set+1):
         $if p == page:
             <span class="this">$p</span>
         $else:
-            <a href="$changequery(page=p)" class="ChoosePage">$p</a>
```

Comment:
I think it would be more useful here to do the offset rather than the page they're clicking on.

So for example if they're on page 1 and click page 2 the offset would be 1.
If they're on page 3 and click page 1 the offset would be -2. This will give us information about how often people are clicking the numbers to skip ahead vs just clicking them to go to the next page.

Reply:
That's a good idea, updated to add this in.